apps/backend/src/resolvers/types/ProposalView.ts

```diff
@@ -112,9 +112,10 @@ export class ProposalResolver {
     @Root() proposal: ProposalView,
     @Ctx() context: ResolverContext
   ): Promise<User | null> {
-    return await context.queries.user.get(
-      context.user,
+    const user = await context.loaders.user.batchLoader.load(
       proposal.principalInvestigatorId
     );
+
+    return user ? user : null;
```

Comment:
I have one question regarding this change. 
I see that `batchLoader.load` gets user list with `getUsersByUserNumbers`, what would happen if we from ess side try to call this query? Does it still work as the same for us?

Reply:
`getUsersByUserNumbers` is implemented only for STFC, for ESS it needs to be implemented. I just added the method signature in `PostgresUserDataSource`, without any implementation.

Reply:
Thank you for the response :)
I'll go ahead and implement the batchLoader on our end and ensure all tests are thoroughly conducted. Once everything checks out, we can then proceed to merge this commit, ensuring success integration for both teams :)